### PR TITLE
just incase

### PR DESCRIPTION
**Pull request by izaakf**
for when things go too far with steely

### CODE REVIEW

**Comment by oskarmcd** [comment]
👍 👍 👍 👍 best plugin since #105

**Comment by izaakf** [comment]
Thank you BRO! I resurrected the egg code as it deserved to be added originally

**Comment by oskarmcd** [comment]
i miss when @sentriz ran this show :-(

**Comment by izaakf** [comment]
@sentriz would've merged

**Comment by sentriz** [comment]
definitely would have merged
bye steely!!!!

**Comment by izaakf** [comment]
they also go CPSSD cancelled

File: steely/plugins/nuke.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+import shutil
+
+
+__author__ = 'izaakf'
+COMMAND = '.nuke'
+
+
+PROBABLY_ENOUGH_TO_BAN_STEELY = 5000
```
Comment (sentriz) [inline steely/plugins/nuke.py]: are you sure this is enough?

Comment (izaakf) [reply steely/plugins/nuke.py]: it may be. If the pr is merged we can properly test, as the local steely instance doesn't get banned by my computer

**Comment by byxor** [comment]
Does this have something to do with Justin Case getting banned last night? ...

**Comment by byxor** [comment]
Did you really all scheme against me to get Justin banned?

**Comment by byxor** [comment]
_really?_

**Pull request closed without merging**